ee/codegen/src/context/workflow-context/workflow-context.ts

```diff
@@ -76,6 +76,8 @@ export class WorkflowContext {
 
   public readonly workflowRawEdges: WorkflowEdge[];
 
+  public readonly outputNamesById: Map<string, string>;
+
```

Comment:
Just left this global tracker here for the output names we use for workflow outputs. Don't know if there is a better spot to put it

Reply:
This makes sense to me! A comment above could help indicate what it's meant to be used for.